app/main.py

```diff
@@ -103,7 +102,7 @@
         dcc.Store(id="last_displayed_event_id", storage_type="memory"),
         dcc.Store(id="images_url_current_alert", storage_type="session", data={}),
         # Storage component which contains data relative to site devices
-        dcc.Store(id="site_devices_data_storage", storage_type="session")
+        dcc.Store(id="site_devices_data_storage", storage_type="session", data=get_site_devices_data(client=api_client))
```

Comment:
So this `dcc.Store` allows one API call once the app is instantiated and then stores it on a session scope right ?

Reply:
Yes, that would be the plan but I am not an expert of these storage components, so I hope it works!